vendor/Browan/AmbientLightSensor-codec.yaml

```diff
@@ -0,0 +1,33 @@
+# Uplink decoder decodes binary data uplink into a JSON object (optional)
+# For documentation on writing encoders and decoders, see: https://thethingsstack.io/integrations/payload-formatters/javascript/
+uplinkDecoder:
+  fileName: AmbientLightSensor.js
+  # Examples (optional)
+  format:
+    - description: Ambient Light Sensor
+      input:
+        fPort: 104
+        bytes: [0, 5]
+      output:
+        data:
+          Status[0]:  
+                Bit [0]       1 - darker, 0 - lighter or not change
+                Bit [1]       1 - lighter, 0 - darker or not change
+                Bits [2~3]    RFU
+                Bit [4]       1 - status change, 0 - not status change
+                Bit [5]       1 - keep-alive, 0 - not keep-alive
+                Bits [6~7]    RFU
```

Comment:
This format is not supported.

The idea here is that this is the output of your codec. So it could be like:

```yaml
output:
  data:
    status: 6
```